.tx/config

```diff
@@ -0,0 +1,8 @@
+[main]
+host = https://www.transifex.com
+
+[<project_slug.resource_slug>]
```

Comment:
I think we need to put a real project_slug and resource_slug here

Reply:
I didn't setup transifex for this project yet. Translations are handmade json files.